src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --light-blue: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
Consider adding a fallback font-family in case the main one doesn't work. You can use a generic font family like 'sans-serif' as the last option.